cli.py

```diff
@@ -19,6 +19,10 @@ def calculate(operation, integers):
     return calc.multiply(integers)
   if (operation == 'average'):
     return calc.average(integers)
+  if (operation == 'tangent'):
```

Comment:
```suggestion
  if (operation == 'tan'):
```

Please look at the issue description. There I have said that usage would be

```sh
python3 cli.py tan 30
```

Reply:
Done